### ISSUE TITLE
Extending Command tooltips with information about State hotkeys

### ISSUE THREAD

**Issue by danylo-dubinin** [ZeroK-RTS/Zero-K#3917]
Hi, I've been augmenting my UI with hotkeys, chaning and experimenting with them a lot to develop an optimal hotkey config for my playstyle. Because of that I sometimes might forget a new hotkey that I'm trying out. 
If it's a **State hotkey**, to recall it I open Epic Menu, click Search, type the name of the action if I remember the name, and find the hotkey in the result list.

I think command tooltips can be extended to cantain more info about configured hotkeys, because we all are sometimes in that moment when you are morphing your geo in 1 match out of 30, and you cannot remember what key you have binded to Low Misc. Priority. I think it's a not neccessary moment of stress so I propose extending the tooltips so they look like in the following examples

- A subset of states has hotkeys.
**Low** doesnt have a hotkey so it's not in the list. *Toggle* is not set, so it's not in the list as well.
![image](https://user-images.githubusercontent.com/20540273/81111284-7bb14b00-8f25-11ea-9aee-a9d83b358b21.png)
Before the change it is empty, nothing is shown at all.

- Only toggle has a hotkey.
Didn't manage to make a screenshot for this one for some reason because I'm totally noob in Windows, but for a state that is only a boolean toggle it says "Toggle: ..." in the same manner as above. If it has another hotkey for On / Off / etc states then it's also listed. 

In short, making all the hotkeys for a State command displayed in the tooltips.
I can submit a PR if you guys find something in it. Criticism is welcome.

**Comment by sprunk** [comment]
>we all are sometimes in that moment when you are morphing your geo in 1 match out of 30, and you cannot remember what key you have binded to Low Misc. Priority.

>Low doesnt have a hotkey so it's not in the list.

Sounds like it would be good to make all of the states show up if any of them has a hotkey (in this example with `Low: (none)`).

**Comment by sprunk** [comment]
6ac84566d15c1d170305c0d375bd3221515ac550 (#3919)
